styles.css

```diff
@@ -103,6 +103,243 @@ header {
   list-style-type: none;
 }
 
+/* Work Section */
+.work {
+  margin-top: 5rem;
+  padding-bottom: 5rem;
```

Comment:
Instead of doing both properties to handle the space, I would recommend sticking to only one, in this case, either of them will accomplish the same :100: , but if you wish to learn more about the usability of these properties, here's a [link that might provide some valuable tips ](https://medium.com/frontendshortcut/margin-vs-padding-c1fc8ea8bfaf) :bulb: 

Besides, you could also accomplish this by using the shorthand property like this:
`margin: 8rem 0;`

**Note:** Kindly increase at least 2rem's to match the distance in the [design you chose](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27). :heavy_check_mark:

Reply:
Hey @karlavdelgadof , Thanks for the positive feedback, I have followed your guidelines and implemented the spacing using the margin shorthand property, in my next pr. Thanks